components/ui/code-block/index.tsx

```diff
@@ -85,6 +91,7 @@ const CodeBlock: React.FC<CodeBlockProps> = ({
             <pre
               className={codeBlockPreVariants({ hasHeader: hasHeader || !roundedTop, breakLines })}
               {...rest}
+              code-block-pre=""
```

Comment:
bruh

Reply:
thank you japan

Reply:
<img width="358" alt="image" src="https://github.com/fiveoutofnine/www/assets/66016924/c613f5f3-abe8-4261-b07f-39f650dee148">